modules/taboolaBidAdapter.js

```diff
@@ -149,9 +153,34 @@ export const spec = {
 
     return bidResponses.map((bidResponse) => getBid(bids, currency, bidResponse)).filter(Boolean);
   },
+  onBidWon: (bid) => {
+    if (bid.nurl) {
+      ajax(replaceAuctionPrice(bid.nurl, bid.originalCpm));
+    }
+  },
+  getUserSyncs: function(syncOptions, serverResponses, gdprConsent, uspConsent) {
```

Comment:
Why not pass the gpp string?

Reply:
@patmmccann 

Didn't know you support gpp and that it's a valid argument for this function (here it is not mentioned at all https://docs.prebid.org/dev-docs/integrate-with-the-prebid-analytics-api.html). 

Now - I pass the gpp string if it exists.

Reply:
it's brand new, nice work